.github/workflows/integration.yml

```diff
@@ -59,8 +69,27 @@ jobs:
               exit 1
           else
             go get github.com/onsi/ginkgo/ginkgo@v1.14.2
-            go mod download
             make ci-e2e
+            sleep 1m # this required to avoid 403 from GitHub API's
```

Comment:
This will be addressed when we fix this issue https://github.com/ossf/scorecard/issues/206